working-groups/WORKING-GROUPS.md

```diff
@@ -192,6 +193,27 @@ Event delivery data plane.
 | -------------------------------------------------------- | ------------------- | ------- | ------------------------------------------------------------------ |
 | <img width="30px" src="https://github.com/matzew.png">   | Matthias Wessendorf | Red Hat | [matzew](https://github.com/matzew)                                |
 
+## Eventing Kafka
+
+A dedidacetd working group for Kafka based Knative Eventing components.
+
+| Artifact                   | Link                                                                                                                     |
+| -------------------------- | -------------------------------------------------------------------------------------------------------------------------|
+| Charter / Mission          | [Charter](/working-groups/kafka/CHARTER.md)                                                                              |
+| Forum                      | [knative-dev@](https://groups.google.com/forum/#!forum/knative-dev)                                                      |
+| Community Meeting VC       | See the top of the [Meeting notes](https://docs.google.com/document/d/1ykBVPtiosGoDjzBklMt9HxbinGrMod6itOvfe6DeVmA/edit) |
+| Community Meeting Calendar | Tuesdays 8:30a-9:00a PST <br>[Calendar](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com)|
+| Meeting Notes              | [Notes](https://docs.google.com/document/d/1ykBVPtiosGoDjzBklMt9HxbinGrMod6itOvfe6DeVmA/edit)                            |
+| Document Folder            | [Folder](https://drive.google.com/drive/folders/1ra5czyKOaMsWuwWXzkhce6_g6rFBdCxg?usp=sharing)                           |
+| Slack Channel              | [#eventing-kafka](https://slack.knative.dev/messages/eventing-kafka)                                                     |
+| Github Team WG leads       | [@knative/kafka-wg-leads](https://github.com/orgs/knative/teams/kafka-wg-leads/members)                                                                       |
+
+| &nbsp;                                                           | Leads           | Company | Profile                                           |
+| ---------------------------------------------------------------- | --------------- | ------- | ------------------------------------------------- |
+| <img width="30px" src="https://github.com/devguyio.png">         | Ahmed Abdalla   | Red Hat | [devguyio](https://github.com/devguyio)           |
+| <img width="30px" src="https://github.com/lionelvillard.png">    | Lionel Villard  | IBM     | [lionelvillard](https://github.com/lionelvillard) |
+| <img width="30px" src="https://github.com/travis-minke-sap.png"> | Travis Minke    | SAP     | [csantanapr](https://github.com/travis-minke-sap) |
+
```

Comment:
not sure what "csantanapr" is but looks like it should be "travis-minke-sap" ?

Reply:
Carlos Santana!

Reply:
My copy paste skills are horrible after 9 pm 😅 sorry lol